### PR TITLE
feat: import Business Trip Regions after install and via patch

### PR DESCRIPTION
**Pull request by barredterra**
Data from https://www.bundesfinanzministerium.de/Content/DE/Downloads/BMF_Schreiben/Steuerarten/Lohnsteuer/2023-11-21-steuerliche-behandlung-reisekosten-reisekostenverguetungen-2024.pdf?__blob=publicationFile&v=1

### CODE REVIEW

**Comment by barredterra** [comment]
@0xD0M1M0 what do you think, is this fine?

**Comment by 0xD0M1M0** [comment]
In principal yes. I am afraid of the maintenance of the list. The values tend to change in the long term. Having the full list gives the user a sense of certainty that everything is correct. If then the list is updated, hundreds of values have to be updated. You could run a patch then, but what happens with the booked trips and their links to the values?

**Comment by barredterra** [comment]
I agree, it's not 100% clean. We'd need to add a validity date.

> You could run a patch then, but what happens with the booked trips and their links to the values?

The booked trips would stay unchanged.

**Comment by barredterra** [comment]
I've added two fields _Valid From_ and _Disabled_, switched to hash naming and made the title non-unique. This way, we can effortlessly add new records and disable the old ones.

File: erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.js
Comment (0xD0M1M0) [inline erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.js:1]: Does this inlcude not showing the disabled ones?

Comment (barredterra) [reply erpnext_germany/erpnext_germany/doctype/business_trip/business_trip.js:1]: Yes, the framework takes care of this by default

**Comment by david-loe** [comment]
@barredterra @0xD0M1M0 I'm maintaining a API for the Pauschbetrag. Maybe this can be helpful: https://github.com/david-loe/pauschbetrag-api

**Pull request merged**